packages/wp-now/src/wp-now.ts

```diff
@@ -67,11 +75,26 @@ export default async function startWPNow(
 	};
 
 	const phpInstances = [];
+
 	for (let i = 0; i < Math.max(options.numberOfPhpInstances, 1); i++) {
 		phpInstances.push(
 			await NodePHP.load(options.phpVersion, nodePHPOptions)
 		);
 	}
+
+	const spawnInstance = async () => {
+		const php = await NodePHP.load(options.phpVersion, nodePHPOptions);
+
+		php.mkdirTree(documentRoot);
+		php.chdir(documentRoot);
+		php.writeFile(
+			`${documentRoot}/index.php`,
+			`<?php echo 'Hello wp-now!';`
+		);
```

Comment:
It looks like a dev artifact, so at a risk of stating the obvious – let's not overwrite the index.php file when spawning a PHP instance.

Reply:
@adamziel https://github.com/WordPress/playground-tools/pull/110/files?diff=unified&w=1#diff-eeed5d391a89564a5ff176b28d0ab5a31e01b94fb75f4ccc1d67f80297586595R85